src/components/search/SearchResultsPaging.jsx

```diff
@@ -69,7 +62,9 @@ const SearchResultsPaging = (props) => {
 
 SearchResultsPaging.propTypes = {
   changePage: PropTypes.func.isRequired,
-  searchType: PropTypes.oneOf(['resource', 'template']).isRequired,
+  totalResults: PropTypes.number.isRequired,
```

Comment:
I'm encountering an error related to a couple of the authorities in the `ld4p:RT:bf2:Monograph:Work:Un-nested` template (from production) in the Subject of Work. When doing a search the `OCLCFAST geographic (QA) - direct` and `OCLCFAST topic (QA) - direct` are causing an error and display a **Nan** for the total records:

![Screen Shot 2020-10-07 at 5 41 10 PM](https://user-images.githubusercontent.com/71847/95399129-b0ef0a00-08c4-11eb-8031-5170a8b3b7c4.png)

From this error in the web console: 
<img width="1171" alt="Screen Shot 2020-10-07 at 5 41 24 PM" src="https://user-images.githubusercontent.com/71847/95399152-be0bf900-08c4-11eb-8712-0bd75e29d14d.png">

I think it is because the returned value from QA for the OCLC FAST direct has a string value in the *total_records* property in header::
```javascript
response_header: {
 start_record: 0, 
 requested_records: "DEFAULT", 
 retrieved_records: 8, 
total_records: "NOT REPORTED"}
```

Reply:
Great catch. I added handling for this.